hooks/useTokens.ts

```diff
@@ -3,73 +3,23 @@ import { PublicKey } from '@solana/web3.js';
 import { useLocalStorage } from '@mantine/hooks';
```

Comment:
Need to remove this file, unless theres a reason to keep it and my logic for moving it into autocrat was flawed.